crates/walrus-service/src/node/dbtool.rs

```diff
@@ -629,6 +653,28 @@ fn read_secondary_slivers(
     Ok(())
 }
 
+fn read_event_processor_init_state(db_path: PathBuf) -> Result<()> {
+    let db = DB::open_cf_for_read_only(&RocksdbOptions::default(), db_path, [INIT_STATE], false)?;
+
+    let Some(cf) = db.cf_handle(INIT_STATE) else {
+        println!("Event processor init state column family not found");
+        return Ok(());
+    };
+
+    let iter = db.iterator_cf(&cf, rocksdb::IteratorMode::Start);
+    for result in iter {
+        let (key, value) = result?;
+        let init_state: InitState = bcs::from_bytes(&value)?;
+        let config = bincode::DefaultOptions::new()
+            .with_big_endian()
+            .with_fixint_encoding();
```

Comment:
nit: I think we can move `config` out of the for loop to only initialize it once?